functions/jsBundle.js

```diff
@@ -20,9 +21,29 @@ const jsHandler = RavenLambdaWrapper.handler(Raven, async event => {
       }
     });
 
-    return createError.InternalServerError(
-      "Could not update the Origami Component list",
-    );
+    if (err.code) {
+      return createError(
+        err.code,
+        `throw new Error(${JSON.stringify(err.message)})`,
+        {
+          headers: {
+            "content-type": "application/javascript;charset=UTF-8",
+            "cache-control": "max-age=0, must-revalidate, no-cache, no-store",
+          },
+        },
+      );
+    } else {
+      // TODO: output stacktrace and error message if not running in production
+      return createError.InternalServerError(
+        `throw new Error(${JSON.stringify("Could not create bundle")})`,
```

Comment:
should we include something like `Origami Build Service returned an error:` in the message to make debugging easier when you see it in the console?

Reply:
That's a good idea!

Reply:
Done! ca751a511d1a10da36b6855892075cb8f0f526b3